app.js

```diff
@@ -157,7 +165,10 @@ function LeftButton(route, navigator, index, navState) {
       return (
         <TouchableOpacity
           style={[styles.backButton, {marginLeft: 10, marginVertical: Platform.OS === 'android' ? 21 : 16}]}
-          onPress={() => navigator.pop()}
+          onPress={() => {
+            navigator.pop()
+            settingsButtonActive = false
```

Comment:
This online function should probably be extracted to a member function,`closeSettingsScreen` or something.